sound/soc/sof/intel/mtl.c

```diff
@@ -728,6 +728,8 @@ int sof_mtl_ops_init(struct snd_sof_dev *sdev)
 
 	ipc4_data->mtrace_type = SOF_IPC4_MTRACE_INTEL_CAVS_2;
 
+	ipc4_data->fw_context_save = true;
+
```

Comment:
commit message

"use this feature to do optimization"

can we clarify what optimizations we are referring to or better change the wording to describe the benefits of this flag?

Reply:
sure, thanks